### DISCUSSION TITLE
How can i add fonts?

### DISCUSSION THREAD

**Discussion by carloshpds** [mozilla/pdf.js#17615] [Q&A]
We're having many problems with multiple types of fonts, standard and non standard:
* Tahoma
* Tahoma-Bold
* CourierNew
* Helvetica-BoldOblique

Can we somehow add these fonts? I would like to redirect these to render with my `OpenSans.ttf`

Configuration:
- Web browser and its version: Brave 1.61.120
- Operating system and its version: MacOS Sonoma 14.2.1
- PDF.js version: 4.0.379
- Is a browser extension: no

Steps to reproduce the problem:
1. Use a pdf generated with any of the fonts listed above

What went wrong? (add screenshot)
![Captura de Tela 2024-02-01 às 14 19 04](https://github.com/mozilla/pdf.js/assets/2482989/f95e6c05-3a4a-4bcf-a590-0bf1ca430b71)

**Comment by sergiubologa** [comment]
@carloshpds have you found the fix to your issue?
